src/settings.json

```diff
@@ -2,8 +2,8 @@
   "calendar": [
     {
       "semester": {
```

Comment:
Let's change this to the date that will never come or to a far future, and update the `vacation_semester` date.